pkgs/servers/asterisk/default.nix

```diff
@@ -90,7 +90,16 @@ let
     sha256 = "1s9idx2miwk178sa731ig9r4fzx4gy1q8xazfqyd7q4lfd70s1cy";
   };
 
-in rec {
+  # auto-generated by update.py
+  versions = lib.mapAttrs (_: {version, sha256}: common {
+    inherit version sha256;
+    externals = {
+      "externals_cache/pjproject-2.10.tar.bz2" = pjproject_2_10;
+      "addons/mp3" = mp3-202;
+    };
```

Comment:
I suppose if we needed it we'd squeeze in an `extraPatches` arg here :point_up:  which could conditionally set itself based on the `version`.